pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -524,6 +524,7 @@ func GetOpenshiftBootstrapClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// push and pull images
 				rbac.NewRule("get", "update").Groups(imageGroup, legacyImageGroup).Resources("imagestreams/layers").RuleOrDie(),
+				rbac.NewRule("create").Groups(imageGroup, legacyImageGroup).Resources("imagestreams").RuleOrDie(),
```

Comment:
it's not clear to me why "update" is not also needed here, and yet we seem to be getting away without it.  Anyone have any theories?

Reply:
the registry does the actual create/update

it gates access to push to existing imagestreams based on `update` `imagestreams/layers`

if the imagestream doesn't exist, it requires the pusher have access to create it

this would allow someone with the image pusher role to create image streams directly. Is that expected?

Reply:
Well, without this it means that if i grant you "image-pusher" access, you can't actually push an image if the imagestream doesn't already exist.  That seems unexpected.

Reply:
(If we want to call this working as designed we can, but it has confused several users on SME and lead to the referenced issue being created)